index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="tool-box">
+        <div class="headder">
+            <h4 class="name">Welcome</h4>
+            <img class="icon" src="images/Union.png" alt="icon">
+        </div>
+    </section>
```

Comment:
[ Required ] Please consider changing this `section` tag to a `header` tag. The semantic `header` is usually used to wrap the logo section and the navigation menu in most websites.

[ Required ] Please consider changing the `h4` tag in this section to an `a` tag, the text `welcome` serves as the logo and in most websites, the logo usually leads to the homepage therefore to improve the navigation of your website you should change it to an `a` tag.

[ Required ] Please consider wrapping the `<img class="icon" src="images/Union.png" alt="icon">` in the semantic html5 `nav` tag, this will improve the accessibility score of your website. Web accessibility tools such as screen readers will easily locate where the navigation menu is in the App.